taca/utils/bioinfo_tab.py

```diff
@@ -83,7 +83,7 @@ def update_statusdb(run_dir):
                             remote_doc = db[remote_id]['values']
                             remote_status = db[remote_id]['status']
                             #Only updates the listed statuses
-                            if remote_status in ['New', 'ERROR', 'Sequencing', 'Demultiplexing', 'QC-Failed', 'BP-Failed', 'Failed']:
+                            if remote_status in ['New', 'ERROR', 'Sequencing', 'Demultiplexing', 'QC-Failed', 'BP-Failed', 'Failed'] and sample_status is not remote_status:
```

Comment:
@sylvinite I suggest to replace the `is not` with `!=` see this post http://stackoverflow.com/questions/1504717/why-does-comparing-strings-in-python-using-either-or-is-sometimes-produce